src/components/QueryCondition.vue

```diff
@@ -117,9 +117,6 @@ export default Vue.extend( {
 				return this.$store.getters.propertyValueRelation( this.conditionIndex );
 			},
 			set( selectedPropertyValueRelation: PropertyValueRelation ): void {
-				if ( selectedPropertyValueRelation === PropertyValueRelation.Regardless ) {
-					this.textInputValue = '';
```

Comment:
Why did you change this?

Reply:
that was not working in the first place. It was throwing an error on the console.
`textInputValue` is not a string (its an object). and I suspect it was put here assuming it will do what this PR is trying to do. 
since its not doing anything, I don't see the purpose of that line of code.

Reply:
I see you may have addressed the issue [here](https://github.com/wmde/query-builder/pull/198/files#diff-216bcf52888451e05bc28ab7fbccc04d1bb8d4181f7bddae00f1ea8fd9f650fdR122)

Reply:
I reverted it. I will merged this one after you merge [198](https://github.com/wmde/query-builder/pull/198)